tests/network/vmi_multus.go

```diff
@@ -1240,6 +1254,10 @@ var _ = SIGDescribe("[Serial]Macvtap", func() {
 				migration := tests.NewRandomMigration(serverVMI.Name, serverVMI.GetNamespace())
 				_ = tests.RunMigrationAndExpectCompletion(virtClient, migration, tests.MigrationWaitTime)
 
+				Eventually(func() error {
```

Comment:
I believe the fix shouldn't be done in the test but in the code itself. In the case of macvtap, migration shouldn't be marked as `MigrationSucceeded` before the source pod is down/the macvtap device is removed.

Let's say for example removing the macvtap device from the original node fails. Any communication between pods/vms on the original node with the migrated vm will fail. Wouldn't it?

Reply:
> Let's say for example removing the macvtap device from the original node fails. Any communication between pods/vms on the original node with the migrated vm will fail. Wouldn't it?

Yes, this is correct. In a longer term I believe we could include the source pod in the MigrationStatus, and add this condition for marking the migration `MigrationSucceeded`.

I'm not sure I have the capacity for this at the moment. Would it be enough for now to extend the test to include explicitly both scenarios:
- client and server on same node before migration
- client and server on different nodes before migration
To have them covered and address the migration condition in a followup?

Reply:
Yes, if we will add XFail with a link to an issue for the "client and server on same node before migration" case it should be enough for now.

Reply:
@AlonaKaplan updated

Reply:
what am i missing ? i don't see XFail in the code (just in this comment and on the PR desc), or it isnt needed anymore ?

Reply:
Not needed. We came to conclusion to wait for the original virt-launcher pod to be terminated.
When it's terminated, the connectivity is not interrupted anymore and traffic should flow from the client vmi to the new virt-launcher pod.

Reply:
Thanks ,can you please update PR desc ?
`For that reason, I'm adding assert.XFail to the ping in the test of scenario 1).`

Reply:
@oshoval sorry for that, Now it's up-to-date